lib/ts/recipe/webauthn/types.ts

```diff
@@ -606,30 +673,17 @@ export type APIInterface = {
               | {
                     status: "OK";
                 }
-              | GenerateRecoverAccountTokenPOSTErrorResponse
               | GeneralErrorResponse
-          >);
-
-    recoverAccountPOST:
-        | undefined
-        | ((input: {
-              webauthnGeneratedOptionsId: string;
-              credential: CredentialPayload;
-              token: string;
-              tenantId: string;
-              options: APIOptions;
-              userContext: UserContext;
-          }) => Promise<
+              //   | GenerateRecoverAccountTokenPOSTErrorResponse
               | {
-                    status: "OK";
-                    email: string;
-                    user: User;
+                    status: "RECOVER_ACCOUNT_NOT_ALLOWED";
+                    reason: string;
                 }
-              | RecoverAccountPOSTErrorResponse
-              | GeneralErrorResponse
+              | { status: "UNKNOWN_USER_ID_ERROR" }
+              | { status: "INVALID_EMAIL_ERROR" }
```

Comment:
Both of these are incorrect statuses to return:
- The input is not a user id, so we can't return UNKNOWN_USER_ID_ERROR
- We do not need to check email validation here cause we are not adding a new email to any user based on this api.
- We only return RECOVER_ACCOUNT_NOT_ALLOWED and OK. In case user with the email is not found, we still return OK cause we do not want to reveal info about user's existing or not. We just dont send an email - similar to what we do in password reset API when we create a new token